### PR TITLE
Quiz parsing

### PR DESCRIPTION
**Pull request by loopiezlol**
- [x] parse **Quiz section** for insights 
- [x] `quiz` rule to check the following format:
```
---
## Quiz

headline: quiz title

question: |
  // This is my question
  // How about some code here
  x = 1;
  x++;
  // what is the result of x

answers:
  - 2
  - 3
  - 4
  - 1 

```

### CODE REVIEW

**Review by stefanoeb — APPROVED** [review]

**Comment by stefanoeb** [comment]
why are there commits on the history that does not appear on the changed files?

**Comment by stefanoeb** [comment]
such as `author db search`

**Comment by loopiezlol** [comment]
<del>umm wanted to do it all together with author db search rule, but by merging the pr it would have made tests on all insight atm break as it the latter was dependent on the PR I just merged on develop (`fix db export bug`) so I'm waiting for a deploy until I merge that<del>

I think I just wanted to separate them, want me to squash this into a cleaner history?

**Pull request merged**
